e2e/schematics/ng-add.test.ts

```diff
@@ -102,7 +103,7 @@ describe('Nrwl Convert to Nx Workspace', () => {
     ).toBeDefined();
     expect(updatedPackageJson.devDependencies['@angular/cli']).toBeDefined();
 
-    const nxJson = readJson('nx.json');
+    const nxJson = readJson<NxJson>('nx.json');
```

Comment:
remove the type.

Reply:
or import it from node_modules (when we run e2e tests we first build nx and copy it into node modules), so you should be able to import it